_inc/client/plans/single-product-backup.jsx

```diff
@@ -0,0 +1,185 @@
+import Gridicon from 'components/gridicon';
+
+export function SingleProductBackup( props ) {
+	const { sitePlans } = props;
+
+	const [ selectedBackupType, setSelectedBackupType ] = useState( 'real-time' );
+
+	return (
+		<>
+			<h1 className="plans-section__header">Single Products</h1>
+			<h2 className="plans-section__subheader">Just looking for backups? We've got you covered.</h2>
+			<div style={ { display: 'flex', justifyContent: 'center', marginBottom: '10px' } }>
+				<AccentedCard>
+					{ {
+						header: <SingleProductBackupHeader sitePlans={ sitePlans } />,
+						body: (
+							<SingleProductBackupBody
+								selectedBackupType={ selectedBackupType }
+								setSelectedBackupType={ setSelectedBackupType }
+								sitePlans={ sitePlans }
+							/>
+						),
+					} }
+				</AccentedCard>
+			</div>
+		</>
+	);
+}
+
+function SingleProductBackupHeader( props ) {
+	const { sitePlans } = props;
+
+	return (
+		<div
+			style={ {
+				display: 'flex',
+				flexDirection: 'row',
+				justifyContent: 'space-between',
+				alignContent: 'center',
+			} }
+		>
+			<h3>{ __( 'Jetpack Backup' ) }</h3>
+			{ sitePlans && (
+				<PlanPriceDisplay
+					dailyPrice={ sitePlans[ 'daily-backup' ].price.yearly.text }
+					yearlyPrice={ sitePlans[ 'realtime-backup' ].price.yearly.amount }
+				/>
+			) }
+		</div>
+	);
+}
+
+export function PlanPriceDisplay( props ) {
+	const { dailyPrice, yearlyPrice } = props;
+	const perYearPriceRange = `${ dailyPrice }-${ yearlyPrice } /year`;
+
+	return (
+		<div
+			style={ {
+				display: 'flex',
+				flexDirection: 'row',
+				justifyContent: 'space-between',
+				alignContent: 'center',
+			} }
+		>
+			<SlashedPrice />
+			<div className="plans-price__container">
+				<span className="plans-price__span">{ perYearPriceRange }</span>
+			</div>
+		</div>
+	);
+}
+
+function SlashedPrice() {
+	return (
+		<div className="slashed-price__container" style={ { marginRight: '14px' } }>
+			<div className="slashed-price__slash"></div>
+			{ /* TODO: get this from an API or calculate, currently unsure how to get this increased price */ }
+			<div className="slashed-price__price">{ '$15-25' }</div>
+		</div>
+	);
+}
+
+function SingleProductBackupBody( props ) {
+	const { selectedBackupType, setSelectedBackupType, sitePlans } = props;
+
+	const upgradeLinks = {
+		'real-time': 'https://wordpress.com/jetpack/connect/pro',
+		daily: 'https://wordpress.com/jetpack/connect/premium',
+	};
+
+	const upgradeTitles = {
+		'real-time': __( 'Upgrade to Real-Time Backups' ),
+		daily: 'Upgrade to Daily Backups',
+	};
+
+	function handleSelectedBackupTypeChange( event ) {
+		setSelectedBackupType( event.target.value );
+	}
+
+	return (
+		<div className="plans-section__body">
+			<p>
+				{ __(
+					'Always-on backups ensure you never lose your site. Choose from real-time or daily backups. {{a}}Which one do I need?{{ext/}}{{/a}}',
+					{
+						components: {
+							a: <a href="https://jetpack.com/upgrade/backup/" />,
```

Comment:
Since we have the external icon, maybe we should add a `target` and `rel` attributes to that link, or use the `ExternalLink` component if we can?

Reply:
I switched this to use ExternalLink.